src/StakingNode.sol

```diff
@@ -66,12 +76,23 @@ contract StakingNode is IStakingNode, StakingNodeEvents, ReentrancyGuardUpgradea
     uint256 public nodeId;
 
     /// @dev Monitors the ETH balance that was committed to validators allocated to this StakingNode
-    uint256 public allocatedETH;
+    uint256 private _unused_former_allocatedETH;
```

Comment:
Question - why not use `unverifiedStakedETH` here? Value could be reinitialized.